x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.test.ts

```diff
@@ -378,6 +404,34 @@ describe('EnterpriseSearchRequestHandler', () => {
     });
   });
 
+  describe('setSessionData', () => {
+    it('sets the value of wsOAuthTokenPackage in a cookie', async () => {
+      const mockNow = 'Thu, 04 Mar 2021 22:40:32 GMT';
+      const mockInAnHour = 'Thu, 04 Mar 2021 23:40:32 GMT';
+      const tokenPackage = 'sEkReTZ';
+
+      const expectedCookie = `${ENTERPRISE_SEARCH_KIBANA_COOKIE}=${tokenPackage}; Path=/; Expires=${mockInAnHour}; SameSite=Lax; HttpOnly`;
+      const sessionDataBody = {
+        _sessionData: { wsOAuthTokenPackage: tokenPackage },
+        regular: 'data',
+      };
+
+      jest.spyOn(global.Date, 'now').mockImplementationOnce(() => {
+        return new Date(mockNow).valueOf();
+      });
```

Comment:
Oh wow, this is a super cool way of not having to manually re-write Date+1. Can I ask that we move this next to the mock dates for clarity? e.g.,

```js
      const tokenPackage = 'some_oauth_secret';

      const mockNow = 'Thu, 04 Mar 2021 22:40:32 GMT';
      const mockInAnHour = 'Thu, 04 Mar 2021 23:40:32 GMT';
      jest.spyOn(global.Date, 'now').mockImplementationOnce(() => {
        return new Date(mockNow).valueOf();
      });
```